thunder/dynamo/utils.py

```diff
@@ -163,6 +163,10 @@ def try_execute_thunder_symbol(thunder_symbol: "Symbol", node: torch.fx.Node) ->
     import thunder
     from thunder.core.trace import TraceCtx
 
+    # `tag_activation_checkpoint` has function input that get_proxy_inputs_from_node can not handle
```

Comment:
We need to support `tag_activation_checkpoint`. What's missing to through get_proxy_inputs_from_node?

Reply:
This is a short cut for now to let the splitter not split for `tag_activation_checkpoint(func, ...)` op (always return True and the splitReason is None), we need to extend this to return the result that whether the `func` is supported by Thunder or not